application/view/omeka/admin/item-set/add.phtml

```diff
@@ -8,38 +8,10 @@ $formLabel = $this->plugin('formLabel');
 $formElement = $this->plugin('formElement');
```

Comment:
Are the various $form helpers still needed on the new versions of the add and edit pages? It doesn't look like they are